dart/utils/sdf/SdfParser.cpp

```diff
@@ -1028,10 +1028,12 @@ void readVisualizationShapeNode(
     const common::Uri& baseUri,
     const common::ResourceRetrieverPtr& retriever)
 {
+  assert(hasAttribute(vizShapeNodeEle, "name"));
```

Comment:
Good call! Changed to print warning and use default name when the name attribute is missing since DART currently doesn't use exceptions for error handling.